WordPress/src/main/res/layout/site_creation_intents_titlebar.xml

```diff
@@ -23,8 +23,7 @@
 
     <Button
         android:id="@+id/skipButton"
-        style="@style/Widget.MaterialComponents.Button.TextButton"
-        android:textColor="@color/blue_50"
+        style="@style/SiteCreationSkipButton"
         android:textSize="@dimen/text_sz_medium"
```

Comment:
Not suggesting a change, but I'm curious if wee need this, now that we've extracted `android:textSize` to `@style/SiteCreationSkipButton` :thinking: ?

Reply:
Good point, I can clean this up 😊 🚀 
Thanks 🥇

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-Android/pull/16994/commits/38654d7d63fceb2b16aaada1133c726d82079637